tools/python/create_reduced_build_config.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import os
+import onnx
+import sys
+
+
+def _extract_ops_from_onnx_graph(graph, operators, domain_opset_map):
+    '''Extract ops from an ONNX graph and all subgraphs'''
+
+    for operator in graph.node:
+        # empty domain is used as an alias for 'ai.onnx'
+        domain = operator.domain if operator.domain else 'ai.onnx'
+
+        if domain not in operators or domain not in domain_opset_map:
+            continue
+
+        operators[domain][domain_opset_map[domain]].add(operator.op_type)
+
+        for attr in operator.attribute:
+            if attr.type == onnx.AttributeProto.GRAPH:  # process subgraph
```

Comment:
> onnx.AttributeProto.GRAPH [](start = 27, length = 26)

Do we also need look at  onnx.AttributeProto.GRAPHS?

Reply:
No ONNX operator currently has an attribute that stores multiple graphs.

Reply:
Maybe some comments here

---
In reply to: [564982253](https://github.com/microsoft/onnxruntime/pull/6428#discussion_r564982253) [](ancestors = 564982253)